typegate/tests/runtimes/wasmedge/wasmedge.py

```diff
@@ -18,3 +28,32 @@ def wasmedge(g: Graph):
             func="add",
         ).with_policy(pub),
     )
+
+
+cwd = sys.argv[1]
+PORT = sys.argv[2]
+gate = f"http://localhost:{PORT}"
+auth = BasicAuth("admin", "password")
+
+wasmedge_tg = wasmedge()
```

Comment:
I suggest using the more generic term `wasm` as all `wasmedge` references will be removed.

Reply:
Definitely, we also need to refactor other places where `wasmedge` is also used.